Source/WebCore/rendering/cocoa/RenderThemeCocoa.h

```diff
@@ -35,10 +35,36 @@ struct AttachmentLayout;
 
 namespace WebCore {
 
+#if ENABLE(FORM_CONTROL_REFRESH)
+
+enum class CornerType {
```

Comment:
We can have tighter packing
```suggestion
enum class CornerType : uint8_t {
```